index.js

```diff
@@ -37,6 +37,15 @@ function isOpen() {
 /**
  * Extension
  */
+function restoreUnreadIndicator() {
+	const indicator = select('.notification-indicator');
+	const status = select('.mail-status', indicator);
+	if (!status.classList.contains('.unread')) {
```

Comment:
would be `contains('unread')` (and not `.unread`), right? :smiley:

Reply:
Ah yeah, I had initially written `status.matchs('.unread')`